doc/contributing.md

```diff
@@ -424,57 +424,54 @@ The core HoloViz packages are also made available on two other conda channels:
-
-```{note}
-Bumping the version of a package depends on the package's nature:
-* Non-pure Python packages, e.g., Panel or GeoViews as they include JavaScript/TypeScript extensions, need their `package.json` and `package-lock.json` files to be updated, manually bumping the version in those files (e.g., from `1.9.5` to `1.9.6-a.1`). Note that the version scheme isn't the same as the Python version scheme.
-* Pure Python packages (and non-pure Python packages once manually bumped) just need a new tag, done with `git tag -m "Version 1.9.6 alpha1" v1.9.6a1 main`.
-```
-
-```{note}
-Development releases are cheap, don't hesitate to make as many as required! It's also fine if a development release only makes it to one of the two distribution platforms (PyPi or Anaconda.org). They're not meant to be for end-users but development purposes only.
-```
-
-```{tip}
-You can tag a release from any branch, not necessarily from the main one. This is useful if you have to maintain multiple versions simultaneously (e.g., 2.* and 3.*).
-```
-
-```{tip}
-If you push a tag by mistake or the wrong tag and are lucky enough to spot that instantly, you should hurry up (really!) and [cancel](https://docs.github.com/en/actions/managing-workflow-runs/canceling-a-workflow) the *Build* and *Documentation* workflows before anything gets published/deployed. If you manage to do that, you can then remove the faulty tag.
-```
+### Before Releasing
+
+Before making a release, consider the following tasks:
+
+- Ensure that decisions regarding release content are reflected in Github Milestones.
+- Confirm the test suite passes successfully.
+- Check for alarming warnings during the test suite execution to avoid missing critical issues.
+  
+### Detailed Process
+
+1. **Development Release (Alpha)**
+   - For non-pure Python packages with JavaScript/TypeScript extensions (e.g., Panel, GeoViews), manually update `package.json` and `package-lock.json` to bump the version (e.g., from `1.9.5` to `1.9.6-a.1`).
+   - For pure Python packages, create a new tag using `git tag -m "Version 1.9.6 alpha1" v1.9.6a1 main`.
+  
+2. **Post-Alpha Checks**
+   - Monitor *Packages* and *Documentation* workflows; fix failures if any.
+   - Verify the new version of the *dev* site and downstream tests on Github Actions.
+
+3. **Beta and Release Candidate (Optional)**
+   - Making *beta* releases is rare; consider a *release candidate* for significant changes.
+   - Announce *release candidate* for user feedback.
+
+4. **Changelog Update**
+   - Update the changelog file (e.g., `CHANGELOG.md`) before the final release with contributions since the last one.
+   - Craft a comprehensive changelog as it aids users in understanding what's new and potential code breakages.
+
+5. **Final Release**
+   - Confirm the last development release's stability, preferably with multiple reviews.
+   - Bump the version in relevant files to the final number (e.g., `v1.9.6`).
```

Comment:
This is unfortunately wrong, we release by creating a tag and pushing it to the repo, not by bumping a version in a file.